package.json

```diff
@@ -50,6 +50,10 @@
         "command": "toggleVim",
         "title": "Vim: Toggle Vim Mode"
       },
+      {
+        "command": "vim.vimTutor",
```

Comment:
In lieu of this, I would suggest implementing support for the command `!vimtutor`. It's more typical of VIM usage and easier to discover.